cmd/dex/config.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	OAuth2     OAuth2      `json:"oauth2"`
 	GRPC       GRPC        `json:"grpc"`
 	Expiry     Expiry      `json:"expiry"`
+	Logger     Logger      `json:"logger"`
```

Comment:
Are there other debug attributes you have in mind that you would like to make part of this struct?